README.md

```diff
@@ -1,13 +1,29 @@
+[![dependencies Status](https://david-dm.org/Austin-Miller-1/react-app-template.svg)](https://david-dm.org/Austin-Miller-1/react-app-template)
+[![devDependencies Status](https://david-dm.org/Austin-Miller-1/react-app-template/dev-status.svg)](https://david-dm.org/Austin-Miller-1/react-app-template?type=dev)
```

Comment:
Is David reliable enough for usage here? Seems to have been down for a while (see issues here: https://github.com/alanshaw/david/issues)